frontend/src/app/page.tsx

```diff
@@ -27,16 +27,21 @@ export default function Home() {
           {videos.map((video) => (
             <li key={video.id}>
               <Link href={`/clips/${video.id}`}>
-                <div className="card card-compact w-96 bg-base-100 shadow-xl">
-                  <figure>
-                    <img src={`http://localhost:8080/api/clips/${video.id}/thumbnail.jpg`} alt="Shoes" />
+                <div className="card card-compact w-96 h-full bg-base-100 shadow-xl min-w-[28rem] min-h-[20rem]">
+                  <figure className={video.processing ? "flex-auto grow self-center pt-8 glass w-full h-full" : ""}>
+                    {video.processing ?
+                      (<div className="">
+                        Processing!
+                      </div>)
+                      : <img src={`http://localhost:8080/api/clips/${video.id}/thumbnail.jpg`} alt="Shoes" />
```

Comment:
```suggestion
                      : <img src={`http://localhost:8080/api/clips/${video.id}/thumbnail.jpg`} />
```

`Alt` tag shouldn't be shoes 🙈 -- Maybe we could do the video title? That could be good for accessibility